docs/content_management/data_migration/exporting_data.md

```diff
@@ -136,9 +136,7 @@ Note that you should test your migrations. See [Importing data](importing_data.m
 
 !!! caution
 
-    `--siteaccess` option usage can be relevant on multi-languages repository.
-    You must export with the SiteAccess supporting all the languages, or migration skips translations in non-supported languages.
-    It is recommended to use the SiteAccess of the Back Office of the targeted repository.
+    [`--siteaccess` option](#siteaccess) usage can be relevant when multiple languages or multiple repositories are used.
```

Comment:
```suggestion
    To prevent data loss, when there are multiple languages or multiple repositories to migrate, make sure to use [`--siteaccess` option](#siteaccess)  to migrate them individually.
```